pysite/base_route.py

```diff
@@ -116,6 +117,9 @@ def error(self, error_code: ErrorCodes) -> Response:
         elif error_code is ErrorCodes.incorrect_parameters:
             data["error_message"] = "Incorrect parameters provided"
             http_code = 400
+        elif error_code is ErrorCodes.database_error:
```

Comment:
I'm not gonna fail your PR for this block because we need to rethink this entire system - but this is inconsistent.